spinalcordtoolbox/registration/algorithms.py

```diff
@@ -341,6 +339,68 @@ def register_step_label(src, dest, step, verbose=1):
     return warp_forward_out, warp_inverse_out
 
 
+def register_rootlet(src, dest, step, ants_registration_params, metricSize, padding, verbose=1):
+    """
+    """
```

Comment:
I was going to suggest writing a docstring, but it looks like none of the functions in this file have docstrings. :sweat_smile: 

Perhaps we can do them all at once in a follow-up PR.